src/range.rs

```diff
@@ -330,17 +330,12 @@ impl Partial {
       XRange::Val(major) => match self.minor {
         XRange::Wildcard => Version {
           major: major + 1,
-          minor: 0,
-          patch: 0,
-          pre: Vec::new(),
-          build: Vec::new(),
+          ..Default::default()
```

Comment:
```suggestion
          minor: 0,
          patch: 0,
          pre: Vec::new(),
          build: Vec::new(),
```

Using `..Default::default()` often creates bugs. I'd prefer to keep this explicit so this code is re-evaluated whenever the properties change even though this is unlikely to ever change.